src/Ploi/Resources/NetworkRule.php

```diff
@@ -40,7 +40,7 @@ public function get(int $id = null): Response
         return $this->getPloi()->makeAPICall($this->getEndpoint());
     }
 
-    public function create(string $name, int $port, string $type = 'tcp', string $fromIpAddress): Response
+    public function create(string $name, int $port, string $type = 'tcp', string $fromIpAddress = ''): Response
```

Comment:
Could you make this default to null instead of a string?

`$fromIpAddress = null`

Reply:
Ok. But what would be the standard on the code?

Because you accepted my other [pull request](https://github.com/ploi-deploy/ploi-php-sdk/pull/20) thats also add a parameter with an empty string instead of `null`.

Reply:
Yes, that should've been null too, I missed that!